de/characters_de.json

```diff
@@ -2276,7 +2276,7 @@
   },
   {
     "id": 27,
-    "name": "Relepathische Ratte",
+    "name": "Telepathische Ratte",
```

Comment:
typo

Reply:
Great. Thanks. I will add it in the next release